frontend/components/bc/input/BcInputNumber.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts" setup generic="WORKAROUND_FOR_CONDITIONAL_PROPS">
+import type { BcInputError } from '~/components/bc/input/BcInputError.vue'
+
+const {
+  labelPosition = 'left', max, min,
+} = defineProps<(
+  {
+    ariaLabel: string,
+    label?: never,
+  } |
+  {
+    ariaLabel?: never,
+    label: string,
+  }
+) & {
+  error?: BcInputError,
+  inputWidth?: `${number}px` | `${number}rem`,
+  labelPosition?: 'left' | 'right',
+  max?: number,
+  min?: number,
+  type?: HTMLInputElement['type'],
+}>()
```

Comment:
suggestion: should we maybe default to `decimal`?
```suggestion
   inputmode?: 'numeric' | 'decimal'
}>()
```

Reply:
Done. And set my usecase in the pricing page to `numeric`